pkg/parser/parser.peg

```diff
@@ -388,7 +388,8 @@ TitleElement <- element:(Word
         / Space+ 
         / CrossReference
         / InlinePassthrough
-        / InlineImage 
+        / InlineIcon
+        / InlineImage
```

Comment:
could you add a few tests just to make sure that inline icons/images can be parsed in title elements?

Reply:
Yes.

Reply:
So it turns out that we only render plain here.  (Just like for images.)  It's possible that this decision could be revisited.  I think the ability to render images and icons inside a title could be useful.  What will happen now is you'll get the Alt text.

Reply:
Ah, I guess that only happens for section 1.

Reply:
Oh, wait.  I guess its for the table of contents only.  Lol.